graphql/schema.graphql

```diff
@@ -259,6 +262,22 @@ type PaymentTerm {
   name: String
 }
 
+enum CustomFieldType {
+  text
+}
+
+type CustomField {
+  name: String
+  type: CustomFieldType
+  value: String
+}
+
+input CustomFieldInput {
+  name: String
+  type: CustomFieldType
```

Comment:
`name` and `type` are required fields so they should have a `!` at the end of the type, eg `name: String!`

We also should have separate types for the settings, passing a `value` is not valid on settings but the current type implies that it is.

Reply:
✅ Created a `SettingsCustomField` for `B2BSettings`